certified-connectors/ZohoSign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1661 @@
+                      },
+                      "description": "actions"
+                    },
+                    "expiration_days": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "Number of days left for the document to get expired.",
+                      "title": "Days to complete",
+                      "x-ms-visibility": "advanced",
+                      "default": 15
+                    },
+                    "is_sequential": {
+                      "type": "boolean",
+                      "description": "Allow recipients to sign in the specified order.",
+                      "title": "Send in order",
+                      "enum": [
+                        true,
+                        false
+                      ],
+                      "default": true
+                    },
+                    "email_reminders": {
+                      "type": "boolean",
+                      "description": "Enable periodic reminders for the recipients to sign the document.",
+                      "title": "Automatic reminders",
+                      "enum": [
+                        true,
+                        false
+                      ],
+                      "x-ms-visibility": "advanced",
+                      "default": true
+                    },
+                    "reminder_period": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "Specify the time interval at which a reminder for signing is to be sent to the recipients.",
+                      "title": "Send a remainder every - (days)",
+                      "x-ms-visibility": "advanced",
+                      "default": 5
+                    },
+                    "folder_id": {
+                      "type": "string",
+                      "description": "folder unique id",
+                      "title": "Folder",
+                      "x-ms-dynamic-values": {
+                        "operationId": "GetFolderList",
+                        "value-path": "folder_id",
+                        "value-collection": "folders",
+                        "value-title": "folder_name"
+                      }
+                    },
+                    "notes": {
+                      "type": "string",
+                      "description": "Notes for all recipients.",
+                      "title": "Notes",
+                      "format": "html"
+                    }
+                  },
+                  "description": "requests",
+                  "required": [
+                    "request_name"
+                  ]
+                }
+              },
+              "required": [
+                "requests"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/requests/{RequestId}/submit": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@ZohoSign-integ Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**